examples/gcp_gpu_direct_tcpx/tcpx_sglang_serving.yaml

```diff
@@ -0,0 +1,50 @@
+name: tcpx-sglang-serving
+
+resources:
+  cloud: gcp
+  instance_type: a3-highgpu-8g
```

Comment:
should we just use `gpus: H100:8` or `gpus: H200:8`

Reply:
Updated